File: app/src/main/java/com/dhsdevelopments/potato/clientapi/channel2/ChannelsResult.java
package com.dhsdevelopments.potato.clientapi.channel2;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class ChannelsResult
{
    @SerializedName( "domains" )
    public List<Domain> domains;

    @Override
    public String toString() {
        return "ChannelsResult[" +
                       "domains=" + domains +
                       ']';
    }
}

File: app/src/main/java/com/dhsdevelopments/potato/clientapi/channel2/ChannelsResult.kt
package com.dhsdevelopments.potato.clientapi.channel2

import com.google.gson.annotations.SerializedName

class ChannelsResult {
    @SerializedName("domains")
    lateinit var domains: List<Domain>

    override fun toString(): String {
        return "ChannelsResult[domains=$domains]"
    }
}
